frontend/src/lib/components/TaxonomicFilter/taxonomicFilterLogic.tsx

```diff
@@ -44,23 +44,28 @@ import { groupDisplayId } from 'scenes/persons/GroupActorHeader'
 import { infiniteListLogicType } from 'lib/components/TaxonomicFilter/infiniteListLogicType'
 import { featureFlagLogic } from 'lib/logic/featureFlagLogic'
 import { FEATURE_FLAGS } from 'lib/constants'
+import { Tooltip } from 'lib/components/Tooltip'
 
-export const eventTaxonomicGroupProps: Pick<TaxonomicFilterGroup, 'getPopupHeader' | 'getIcon'> = {
-    getPopupHeader: (eventDefinition: EventDefinition): string => {
-        if (!!keyMapping.event[eventDefinition.name]) {
-            return 'Verified Event'
-        }
-        return `${eventDefinition.verified ? 'Verified' : 'Unverified'} Event`
-    },
-    getIcon: getEventDefinitionIcon,
+export function createEventTaxonomicGroupProps(
+    shouldSimplifyActions: boolean = false
```

Comment:
instead of passing this `shouldSimplifyActions` param everywhere in the other calls how come we don't we remove it and just call the value once from the feature flags?

Reply:
it'll prob reduce clean up work later on

Reply:
not blocking though

Reply:
talked offline - might be difficult to mount logic inside this util since it doesn't return a jsx element

should be fine if i cant find an easy refactor for this